woof-code/3builddistro

```diff
@@ -493,6 +493,23 @@ for i in $(ls sandbox3/rootfs-complete/usr/share/locale); do
 	case $i in en|en_US|locale.alias) continue ;; esac
 	rm -rf sandbox3/rootfs-complete/usr/share/locale/$i
 done
+
+#overwrite current documentation if DOCS_URL present
+if [ -n "$DOCS_URL" ]; then
+	echo "Overwriting current documentation with developer provided $DOCS_URL..."
+	rm -rf sandbox3/rootfs-complete/usr/share/doc
+
+	[ -f "../../local-repositories/$DOCS_URL.doc.zip" ] || wget_output=$(wget -t 3 -T 60 -O "../../local-repositories/$DOCS_URL.doc.zip" "$DOCS_URL")
```

Comment:
The correct way to check if wget has failed is:

```
	wget -t 3 -T 60 -O "../../local-repositories/$DOCS_URL.doc.zip" "$DOCS_URL"
	if [ $? -ne 0 ]; then
```

(exit code, not output)

Reply:
Does it check with output? I thought `$()` returned exit code. Will backticks (`) return exit code then?

(Sorry, I am new to bash, actually am a programmer of C.)